src/widgets/immunizations/immunizations-detailed-summary.component.tsx

```diff
@@ -159,13 +159,13 @@ export default function ImmunizationsDetailedSummary(
       >
         <div className={styles.immunizationMargin}>
           <p className="omrs-medium">
-            <Trans i18nKey="no immunization configured">
-              No Immunizations are configured.
+            <Trans i18nKey="noImmunizationsAreConfigured">
+              No immunizations are configured.
             </Trans>
           </p>
           <p className="omrs-medium">
             <a href="https://github.com/openmrs/openmrs-esm-patient-chart-widgets#configuration">
-              <Trans i18nKey="please configure immunization">
+              <Trans i18nKey="pleaseConfigureImmunizations">
```

Comment:
This copy seems likely to change. How about

```suggestion
              <Trans i18nKey="configureImmunizationsPrompt">
```